### DISCUSSION TITLE
Nested NuxtLinks on different pages

### DISCUSSION THREAD

**Discussion by NortjeDanie** [nuxt/nuxt#25584] [Questions]
By using two NuxtLinks on different pages (For example: A NuxtLink on a homepage that Navigates to a product page which also contains a NuxtLink) I keep getting the following error: 

[nuxt] [NuxtLink] You can't nest one <a> inside another <a>. This will cause a hydration error on client-side. You can pass the `custom` prop to take full control of the markup.

When passing the custom prop the Link the error is resolved but does not rout to my product page.

Any answers on my struggles?

**Comment by AloisSeckar** [comment]
Can you post some code snippet? Or provide minimal reproduction via https://stackblitz.com/github/nuxt/starter/tree/v3-stackblitz?file=README.md ?

The warning sounds like you are trying to wrap whole product page into NuxtLink, which really isnt the best idea.
